### PR TITLE
View Pages (Profiles)

### PR DESCRIPTION
**Pull request by MatthewLacerda2**
A new section for the website for viewing Profile pages of Users and Real Estates

Internally they are called "View" pages to avoid coding mis-interpretation. As far as the user is concerned, those are "Profiles".

These pages are using the API, however i had to remove the Authorization for the HttpClient to pass. I'll look into passing the Headers properly

Note: the API does not search filter RealEstates to those owned by a specific Company yet. This will be addressed when filtering is made automatic with [JsonApi.Net](https://www.jsonapi.net/)

### CODE REVIEW

**Review by pgbezerra — APPROVED** [review]

File: webserver/Controllers/CompanyController.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 namespace webserver.Controllers;
 
-[Authorize(Roles=Common.BZE_Role+","+Common.Company_Role)]
```
Comment (pgbezerra) [inline webserver/Controllers/CompanyController.cs]: If you don't need them, drop them.
But I think they are necessary. We don't want unauthenticated users to access this controller, do we?

File: webserver/Controllers/RealEstatesController.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 namespace webserver.Controllers;
 
-[Authorize(Roles=Common.BZE_Role+","+Common.Company_Role)]
+//[Authorize(Roles=Common.BZE_Role+","+Common.Company_Role)]
```
Comment (pgbezerra) [inline webserver/Controllers/RealEstatesController.cs]: same question here

File: webserver/Pages/Manage/ClientCRUD.cs [hunk 3]
```diff
@@ -48,9 +48,10 @@ public class ClientCRUD : PageModel {
...
-}
-public class ClientINFO{
-    public string Id { get; set; } = string.Empty;
-    public string UserName { get; set; } = string.Empty;
-    public string Occupation{get; set; } = string.Empty;
+
+    public class ClientINFO{
+        public string Id { get; set; } = string.Empty;
+        public string UserName { get; set; } = string.Empty;
+        public string Occupation{get; set; } = string.Empty;
```
Comment (pgbezerra) [inline webserver/Pages/Manage/ClientCRUD.cs:55]:
```suggestion
        public string Occupation{ get; set; } = string.Empty;
```

You need to setup an auto formatted in your IDE.

**Review by pgbezerra — DISMISSED** [review]

**Review by pgbezerra — COMMENTED** [review]
Sorry, I approved, but there are open questions.

File: webserver/Controllers/CompanyController.cs (hunk 1 above)
Comment (MatthewLacerda2) [reply webserver/Controllers/CompanyController.cs]: Yeah i probably commented it out when testing/debugging. We do need it, so it's back now

**Pull request merged**